docs/reference/glossary.asciidoc

```diff
@@ -61,6 +61,15 @@
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is a non-scoring <<glossary-query,query>>, meaning that it does not score documents.
+  It is only concerned about answering the question - "Does this document match?". 
+  The answer is always a simple, binary yes or no. This kind of query is said to be made 
+  in a https://www.elastic.co/guide/en/elasticsearch/guide/current/_queries_and_filters.html[filter context], 
```

Comment:
I think I see your problem now.
You are trying to link to a page in the elasticsearch guide, which only seems to work with an [external link syntax](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links). What I said earlier was to please use an "Internal cross references" as described in the asciidoc syntax above to point to [this document](https://github.com/elastic/elasticsearch/blob/master/docs/reference/query-dsl/query_filter_context.asciidoc) in the reference itself, which works because of its inline anchor in the first line. You should be able to use `<<query-filter-context,filter context>>` for this.

Reply:
I have tried even this `<<query_filter_context.asciidoc,filter context>>` before, now again, it is not working. It only works if both the linked and linking documents are in the same folder. Here we have `docs/reference/glossary.asciidoc` linking to `docs/reference/query-dsl/query_filter_context.asciidoc`.  I found this [issue](https://github.com/asciidoctor/asciidoctor/issues/844) on asciidoctor repo but I could not make anything of it. I tried the follow with no success :- 
`<</query-dsl/query_filter_context.asciidoc,filter context>>`  
`<<./query-dsl/query_filter_context.asciidoc,filter context>>` 
 `<<../query-dsl/query_filter_context.asciidoc,filter context>>`

I build this before pushing and it did not break any test. Must all references follow the `<<,>>` syntax ? Sorry, please I don't seem to get the problem, in this case.

Reply:
This `<<query-filter-context,filter context>>` gives me this:
```
/temp/asciidoc/index.xml:78053: element link: validity error : IDREF attribute linkend references an unknown ID "query_filter_context"
```

I am on Windows 7, using CGwin - bash, perl, python, xsltproc installed

![elasticdocs](https://user-images.githubusercontent.com/17991837/38734284-4767238a-3f25-11e8-8509-335140cda494.PNG)

Reply:
Can you please push the version where you get that error?